src/components/settings/SettingsLMSTab/tests/LmsConfigPage.test.jsx

```diff
@@ -217,6 +225,33 @@ describe('<SettingsLMSTab />', () => {
     expect(screen.queryByText('Exit without saving')).toBeFalsy();
     expect(screen.queryByText('Connect SAP')).toBeFalsy();
   });
+  test('Expected behavior when customer has no IDP configured', async () => {
+    const expectedApiResponse = {
+      data: { needs_idp_config: true },
+    };
+    mockNeedsIDPConfig.mockImplementation(() => (Promise.resolve(expectedApiResponse)));
+    const history = createMemoryHistory();
+    render(
+      <Router history={history}>
+        <Provider store={mockStore({ ...initialState })}>
+          <SettingsLMSTab enterpriseId={enterpriseId} enterpriseSlug={enterpriseSlug} />
+        </Provider>
+      </Router>,
+    );
+    expect(history.location.pathname).toEqual('/');
+    await screen.findByText('Configure SSO');
+    const configureSSOButton = screen.getByText('Configure SSO');
+    fireEvent.click(configureSSOButton);
+    expect(history.location.pathname).toEqual(`/${enterpriseSlug}/admin/settings/sso`);
+  });
+  test('Expected behavior when customer has IDP configured', async () => {
+    const expectedApiResponse = {
+      data: { needs_idp_config: false },
+    };
+    mockNeedsIDPConfig.mockImplementation(() => (Promise.resolve(expectedApiResponse)));
+    renderWithRouter(<SettingsLMSWrapper />);
+    expect(screen.queryByText('No SSO configured')).toBeFalsy();
```

Comment:
I would recommend using the jest-dom extended matchers such as `.not.toBeInTheDocument()` for verifying dom state, since there are cases wheren things don't render and yet toBeTruthy and toBeFalsy can be still true/false

Reply:
I was just following conventions found in the admin portal! I can easily change.